examples.py

```diff
@@ -18,67 +19,66 @@
-import numpy as np
+# from matplotlib import colors as mcolors
 
-# Fixing random state for reproducibility
-np.random.seed(19680801)
+# # Fixing random state for reproducibility
+# np.random.seed(19680801)
 
 
-def polygon_under_graph(xlist, ylist):
-    """
-    Construct the vertex list which defines the polygon filling the space under
-    the (xlist, ylist) line graph.  Assumes the xs are in ascending order.
-    """
-    return [(xlist[0], 0.), *zip(xlist, ylist), (xlist[-1], 0.)]
+# def polygon_under_graph(xlist, ylist):
+#     """
+#     Construct the vertex list which defines the polygon filling the space under
+#     the (xlist, ylist) line graph.  Assumes the xs are in ascending order.
+#     """
+#     return [(xlist[0], 0.), *zip(xlist, ylist), (xlist[-1], 0.)]
 
 
-fig = plt.figure()
-ax = fig.gca(projection='3d')
+# fig = plt.figure()
+# ax = fig.gca(projection='3d')
 
-# Make verts a list, verts[i] will be a list of (x,y) pairs defining polygon i
-verts = []
-# verts = [L]
+# # Make verts a list, verts[i] will be a list of (x,y) pairs defining polygon i
+# verts = []
+# # verts = [L]
 
 
-# Set up the x sequence
-xs = np.linspace(0., 10., 1000)
+# # Set up the x sequence
+# xs = np.linspace(0., 10., 1000)
 
-# The ith polygon will appear on the plane y = zs[i]
-# zs = range(4)
+# # The ith polygon will appear on the plane y = zs[i]
+# # zs = range(4)
 
-# for i in zs:
-#     ys = np.random.rand(len(xs))
-#     verts.append(polygon_under_graph(xs, ys))
+# # for i in zs:
+# #     ys = np.random.rand(len(xs))
+# #     verts.append(polygon_under_graph(xs, ys))
 
-# poly = PolyCollection(verts, facecolors=['r','g','b','r'], linewidths=3, alpha=.6)
-# ax.add_collection3d(poly, zs=zs, zdir='y')
-poly = PolyCollection(L,
-                      linewidths =[0.5 for _ in range(len(L))],
-                      edgecolors=['k' for _ in range(len(L))],
-                     # facecolors=plt.cm.jet,
-                      facecolors=['g','g','b','r','g','y','b','r','g'],
-                      alpha=.7)
-ax.add_collection3d(poly, zs=0.2*np.linspace(0.,10.,len(L)), zdir='y')
+# # poly = PolyCollection(verts, facecolors=['r','g','b','r'], linewidths=3, alpha=.6)
+# # ax.add_collection3d(poly, zs=zs, zdir='y')
+# poly = PolyCollection(L,
+#                       linewidths =[0.5 for _ in range(len(L))],
+#                       edgecolors=['k' for _ in range(len(L))],
+#                      # facecolors=plt.cm.jet,
+#                       facecolors=['g','g','b','r','g','y','b','r','g'],
+#                       alpha=.7)
+# ax.add_collection3d(poly, zs=0.2*np.linspace(0.,10.,len(L)), zdir='y')
 
-#ax.plot(L,np.linspace(0.,1.,len(L)))
+# #ax.plot(L,np.linspace(0.,1.,len(L)))
 
-ax.set_xlabel('X')
-#ax.set_ylabel('Y')
-ax.set_zlabel('Z')
-ax.set_xlim(0, 0.3)
-ax.set_ylim(-1, 4)
-ax.set_zlim(0, 0.06)
+# ax.set_xlabel('X')
+# #ax.set_ylabel('Y')
+# ax.set_zlabel('Z')
+# ax.set_xlim(0, 0.3)
+# ax.set_ylim(-1, 4)
+# ax.set_zlim(0, 0.06)
```

Comment:
I just wanted to practice commenting. Is this the part the graphs the black outline?

Reply:
No, that sets the x, y, and z ranges of the axes drawn, together with the label that is displayed next to them.